vllm/v1/engine/async_llm.py

```diff
@@ -444,7 +445,7 @@ async def get_model_config(self) -> ModelConfig:
         return self.model_config
 
     async def get_decoding_config(self):
```

Comment:
Should we rename this method?

Reply:
I don't think we use this function for anything atm

fwiw we can always get this from `get_vllm_config().structured_outputs_config`

Reply:
I'm not sure why these getters exist tbh. Do you need async getters to access members of classes used in async contexts?

Reply:
I can't find anywhere that we use this, will probably create a smaller PR to remove these noop